GUI/Forms/ExtractProgressForm.cs

```diff
@@ -246,6 +246,7 @@ private async Task ExtractSubfiles(string contentRelativeFolder, ContentFile con
         {
             foreach (var contentSubFile in contentFile.SubFiles)
             {
+                cancellationTokenSource.Token.ThrowIfCancellationRequested();
```

Comment:
FWIW a good place to add one of these, but doesn't fully fix the issue. Need to pass in the cancellation token into gltf exporter itself  and place a few throws in there too. Because exporting something huge will still take a very long time (e.g. map adding many models, or writing textures to disk).

Reply:
> Need to pass in the cancellation token into gltf exporter itself and place a few throws in there too.

Done, but ExportFile#Export has no cancellation token to pass to glTF exporter, so exporting single file (i.e. world) will still not cancel properly.

Reply:
That method should probably be updated to use ExtractProgressForm for consistency, but that can be done in a future PR.

Reply:
Yeah I don't really want to dig into GUI code, so I'd leave it for someone else. 

Removed `Fixes #470` from commit description, also sorry for the reference spam caused by force-pushes.

Reply:
Easiest path to have cancellation token there would be making `GenericProgressForm` `cancellationTokenSource` public.